types/node/tty.d.ts

```diff
@@ -45,7 +45,7 @@ declare module "tty" {
         /**
          * Moves this WriteStream's cursor to the specified position.
          */
-        cursorTo(x: number, y: number, callback?: () => void): boolean;
+        cursorTo(x: number, y?: number, callback?: () => void): boolean;
```

Comment:
please add also `cursorTo(x: number, callback?: () => void): boolean`.

Reply:
Done, and added a test too.